RepoDb.Core/RepoDb.Tests/RepoDb.IntegrationTests/Models/IdentityTableWithDifferentPrimary.cs

```diff
@@ -1,16 +1,24 @@
-﻿using RepoDb.Attributes;
-using System;
+﻿using System;
+using System.Globalization;
+using RepoDb.Attributes;
+using RepoDb.Entity;
 
 namespace RepoDb.IntegrationTests.Models
 {
     [Map("[sc].[IdentityTable]")]
-    public class IdentityTableWithDifferentPrimary
+    public class IdentityTableWithDifferentPrimary : BaseEntity<IdentityTableWithDifferentPrimary>
```

Comment:
Thinking to really avoid the base class again - it has been removed before and I will not reconsidering to putting it back. Here is the link of [removal](http://repodb.net/release/core/repodb-v135).

Reply:
I understand your decision for not wanting to have the base class.  Thinking about it, it does bind the object to the ORM.

Reply:
I certainly understand your reasoning. The base class binds the DTOs to the ORM.  This is one of the issues with EF DTO's.